packages/rpc/README.md

```diff
@@ -0,0 +1,6 @@
+[//]: # `{ "title": "@mintbase-js/rpc", "order": 1 }`
```

Comment:
The order here is how gitbook presents the menu, probably want to sort this closer to testing I think 6 or 7.  You can preview how the docs look by checking this PR https://github.com/Mintbase/gitbook-docs/pull/7

Reply:
Done